Floofbot/Modules/Tag.cs

```diff
@@ -7,191 +7,188 @@
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 
-namespace Floofbot
+namespace Floofbot.Modules
 {
-    public partial class Utility
+    [Group("tag")]
+    [RequireContext(ContextType.Guild)]
+    [RequireUserPermission(Discord.GuildPermission.AttachFiles)]
+    public class Tag : ModuleBase<SocketCommandContext>
     {
-        [Group("tag")]
-        [RequireContext(ContextType.Guild)]
-        [RequireUserPermission(Discord.GuildPermission.AttachFiles)]
-        public class Tag : ModuleBase<SocketCommandContext>
+        private SqliteConnection dbConnection;
+        private static readonly Discord.Color embedColor = Color.Magenta;
+
+        public Tag()
         {
-            private SqliteConnection dbConnection;
-            private static readonly Discord.Color embedColor = Color.Magenta;
-
-            public Tag()
-            {
-                dbConnection = new SqliteConnection(new SqliteConnectionStringBuilder {
-                    DataSource = "botdata.db"
-                }.ToString());
-                dbConnection.Open();
-            }
+            dbConnection = new SqliteConnection(new SqliteConnectionStringBuilder {
+                DataSource = "botdata.db"
+            }.ToString());
+            dbConnection.Open();
+        }
```

Comment:
Use entity framework not sqlite

Reply:
I'd prefer handling this in a future PR that deals with the models and/or the entity framework integration.